src/mower_logic/src/monitoring/monitoring.cpp

```diff
@@ -207,14 +225,7 @@ void reconfigCB(const mower_logic::MowerLogicConfig &config) {
   ROS_INFO_STREAM("Monitoring received new mower_logic config");
   mower_logic_config = config;
 
-  // Set and publish new sensor limits
-  for (auto &sc_pair : sensor_configs) {
-    // Skip if sensor doesn't exists
-    if (sc_pair.second.existCB != nullptr && !sc_pair.second.existCB()) continue;
-
-    set_sensor_limits(sc_pair.second);
-    sc_pair.second.si_pub.publish(sc_pair.second.si);
-  }
+  registerSensors();
```

Comment:
`registerSensors()` seems to do much more than the previous code. Maybe introduce a parameter to call a "lite" version of it?